Dockerfile

```
@@ -1,15 +1,15 @@
-FROM alpine:3.12 as builder
+FROM alpine:3.20 as builder
 RUN apk add --no-cache build-base cmake ninja python3 \
       binutils-dev curl-dev elfutils-dev
 WORKDIR /root
-ENV KCOV=https://github.com/SimonKagstrom/kcov/archive/v38.tar.gz
+ENV KCOV=https://github.com/SimonKagstrom/kcov/archive/refs/tags/v42.tar.gz
 RUN wget -q $KCOV -O - | tar xz -C ./ --strip-components 1
 RUN mkdir build && cd build \
  && CXXFLAGS="-D__ptrace_request=int" cmake -G Ninja .. \
```

```suggestion
 && cmake -G Ninja .. \
```
For the next release https://github.com/SimonKagstrom/kcov/pull/443